db/data_migration/20150716094419_add_eu_logo_to_regional_development_funding.rb

```diff
@@ -0,0 +1,25 @@
+slugs = [
+  'erdf-programmes-and-resources',
+  'erdf-programmes-progress-and-achievements',
+  'applying-for-erdf-funding',
+  'erdf-national-guidance',
+  'previous-erdf-programmes-retaining-documents',
+  'european-territorial-cooperation-programmes',
+  'reuniting-europe-programme-turkey'
+]
+logo_url = "https://assets.digital.cabinet-office.gov.uk/government/assets/erdf-logo.png"
```

Comment:
We shouldn't really be using this URL in the database. This url is dependant on an image being in checked into this repository. If we were to remove this asset (because as of now nothing calls it so there is no reason to keep the asset around) this will break. Where we are doing this [in other places](https://github.com/alphagov/whitehall/commit/140e5ef8db207a436ee2c396c83ece3989504c15#diff-652cafe5973b86054485c529b74b4a07R1) we have uploaded an asset using asset-manager. As assets are kept forever so they can be used in content it won't go away when someone cleans up a repository.